README.md

```diff
@@ -217,26 +219,21 @@ dokku docker-options:add ckan run,deploy --link ckan-solr.web.1:solr
 dokku docker-options:add ckan run,deploy --link ckan-datapusher.web.1:ckan-datapusher
 ```
 
-### Configure Database
-
-Before you can run CKAN for the first time, you need to run `db init` to initialize your database
+### Scheduled Jobs
+For OpenAfrica to work perfectly, some jobs have to run at certain times e.g. updating tracking statistics and rebuilding the search index for newly uploaded datasets. To create a scheduled job that is executed by a Dokku application, follow these steps:
 
-```
-dokku enter ckan
-cd src/ckan
-paster db init -c /ckan.ini
+```sh
+sudo su
+su dokku
+crontab -e
 ```
 
-### Scheduled Jobs
-For OpenAfrica to work perfectly, some jobs have to run at certain times e.g. updating tracking statistics and rebuilding the search index for newly uploaded datasets. These jobs are setup in crontab
+Add the following entries
 
 ```sh
-@hourly echo '{}' | /usr/lib/ckan/default/bin/paster --plugin=ckan post -c /etc/ckan/default/production.ini /api/action/send_email_notifications > /dev/null
-
-@hourly /usr/lib/ckan/default/bin/paster --plugin=ckan tracking update -c /etc/ckan/default/production.ini
-@hourly /usr/lib/ckan/default/bin/paster --plugin=ckan search-index rebuild -r -c /etc/ckan/default/production.ini
+0 * * * * echo '{}' | dokku --rm run ckan paster --plugin=ckan post -c /ckan.ini /api/action/send_email_notifications > /dev/null
 
-*/15 *  *   *   *     /usr/lib/ckan/default/bin/paster --plugin=ckanext-harvest harvester run --config=/etc/ckan/default/production.ini
+0 * * * * dokku --rm run ckan paster --plugin=ckan tracking update -c /ckan.ini
```

Comment:
What about for harvesting?

Reply:
@DavidLemayian there's nothing set up for harvesting, the harvester plugin is not even enabled. I might be wrong but I thought it was a redundant job